Gemfile

```diff
@@ -42,7 +42,7 @@ gem "gettext_i18n_rails",               "~>1.7.2"
 gem "gettext_i18n_rails_js",            "~>1.3.0"
 gem "hamlit",                           "~>2.11.0"
 gem "inifile",                          "~>3.0",             :require => false
-gem "inventory_refresh",                "~>1.0",             :require => false
+gem "inventory_refresh",                "~>1.1",             :require => false
```

Comment:
> Placeholder to cross-repo test inventory_refresh master (going to be v2.0)

Is this currently just testing with 1.1.x as a baseline for cross-repo and will be updated to 2.0 once that's ready to go? I can't tell if you meant to test 2.0 but instead had 1.1

Reply:
Exactly, description :point_up: 

> Placeholder to cross-repo test inventory_refresh master (going to be v2.0)

Reply:
Ok, cool, I'll leave the failing test on the ruby 3 branch for now and will drop my inventory refresh Gemfile change once this gets in.  Thanks for the clarification.